src/views/postcode-registered-office-address.njk

```diff
@@ -65,13 +65,14 @@
           }) }}
 
           <p>
-            <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterRegisteredOfficeAddressPageType) }}" class="govuk-body-m govuk-link" data-event-id="enter-address-manually">{{ i18n.officeAddress.enterAddressManually }}</a>
+            <a href="{{ props.currentUrl | replace(props.pageType, props.data.enterRegisteredOfficeAddressPageType) }}" class="govuk-body-m govuk-link" onclick="_paq.push(['trackEvent', 'Limited Partnerships registered addess', 'manual-page-link', 'enter-address-manually']);">{{ i18n.officeAddress.enterAddressManually }}</a>
```

Comment:
the previous data-event-id is working ok, just tested on main branch
<img width="784" alt="Screenshot 2025-02-04 at 13 05 53" src="https://github.com/user-attachments/assets/2339ac79-dbf9-40fd-bb15-bf4d17d4f2d4" />

Reply:
Changed back to data-event id.